src/OfficeToolPlus/Dictionaries/Languages/de-de.xaml

```diff
@@ -114,7 +114,7 @@
     <sys:String x:Key="Deploy_Deployment">Office bereitstellen</sys:String>
     <sys:String x:Key="Deploy_Deployment.Description">Stellen Sie vor der Bereitstellung von Office sicher, dass der Bereitstellungsmodus korrekt eingestellt ist und das Produkt und die Sprache konfiguriert sind.</sys:String>
     <sys:String x:Key="Deploy_Start_Deploy">Office _bereitstellen</sys:String>
-    <sys:String x:Key="Deploy_Reset">Alle Einstellungen zurü_cksetzen</sys:String>
+    <sys:String x:Key="Deploy_Reset">Alle Einstellungen _zurücksetzen</sys:String>
```

Comment:
The underscore must remain before "c" or "r", because "z" is already in use at "Deploy_View_Code".
Should be "zu_rücksetzen".

Reply:
Was bedeutet der Strich eigentlich, sorry für meine ... Frage.
What does the dash actually mean, sorry for my ... question.

Reply:
`_`
Underline the next character. Used when pressing ALT on the keyboard as an alternative to hotkeys (keystroke).
![image](https://github.com/YerongAI/Office-Tool/assets/19782385/08f99d74-2552-4687-9808-015572b8dfa4) -> ![image](https://github.com/YerongAI/Office-Tool/assets/19782385/010abacb-97b3-442d-a9d8-50e6a5706ffb)

There is a [similar option](https://support.microsoft.com/office/95ef89dd-7142-4b50-afb2-f762f663ceb2) in Microsoft Office.
![image](https://github.com/YerongAI/Office-Tool/assets/19782385/a861ca21-81d2-40c0-a108-a2681e68d781)